org/org.yaml

```diff
@@ -277,3 +277,399 @@ orgs:
         privacy: closed
         repos:
           triggers: write
+      website.maintainers:
+        description: the website maintainers
+        maintainers:
+        - bobcatfish
+        members:
+        - bobcatfish
+        - dlorenc
+        - vdemeester
+        - kimsterv
+        - abayer
+        - lucperkins
+        - afrittoli
+        - skaegi
+        - AlanGreene
+        - jjasghar
+        - popcor255
+        privacy: closed
+        repos:
+          website: write
+      core.collaborators:
+        description: The core collaborators
+        maintainers: !!set
```

Comment:
Any reason on the `!!set` ?

Reply:
Oh because it is a list for the `maintainers`

Reply:
uhm, yeah, but I need to fix this.